### DISCUSSION TITLE
Remove achievements completely

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#115897] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In response to https://github.com/orgs/community/discussions/106536 I'm opening this discussion to propose the complete removal of the Achievement system itself.

There are many reasons, but these here are the ones I consider the most prominent and important to consider:

### Achievements encourage low-effort, nonsensical contributions and actions.

What happens in these discussions also can happen on any other repository of any other user, organization, community... So blocking gaining it here won't really stop it, but only move it away to other targets, causing them the stress of dealing with the spam.

### Achievements make contributions more competitive, rather than collaborative.

People get encouraged to contribute, which may sound great at first, but this may often end up in them doing it in a more competitive nature, rather than in a team-like effort.
Instead on working on an issue in a project collectively, one may just try to do it themself to "gain" points for the achievement.
This goes against what Open source Projects should be about: Fostering a community of people helping one another building better projects.

### Achievements are 100% pointless account padding

They bring nothing to the table of a user. They don't unlock new stuff, they don't put you on some leaderboard. They are just a cosmetic that does nothing other than taking up space on your account.
And while some of you may know that, a majority of people may not. And this may cause them to try and get those achievements, assuming it gives them something in return, making this a situation where someone makes contributions out of a sense for greed, rather than out of a sense for helping.

**Comment by CopperEagle** [comment]
Thank you for your thoughts! The main point of this platform really *is* collaboration, besides being a cloud. So we should focus more on enhancing this aspect :+1:

**Comment by ViggoV** [comment]
I couldn't agree more!

**Comment by vishveshjain** [comment]
Thank you for sharing your thoughts on the achievement system! It’s an interesting topic, and I appreciate your perspective. Let’s break down the points you’ve raised:

Low-Effort Contributions:
Achievements can indeed encourage low-effort contributions. When the focus is solely on earning achievements, some users might prioritize quantity over quality.
Blocking achievements in one place could lead to similar behavior elsewhere, as you mentioned. It’s a challenge to strike the right balance.
Competitiveness vs. Collaboration:
You’re absolutely right that achievements can sometimes shift the focus from collaboration to competition. Ideally, open-source projects thrive on teamwork and collective effort.
When individual achievement becomes the primary motivation, it might hinder collaboration and shared problem-solving.
Pointless Account Padding:
The cosmetic nature of achievements is an interesting point. While some users appreciate them, others might find them irrelevant.
Educating users about the true purpose of achievements could help manage expectations and prevent misguided efforts.
Overall, achieving a balance between encouraging contributions and maintaining a collaborative spirit is crucial. Perhaps refining the achievement system or providing clearer guidelines could address some of these concerns.
